code/modules/antagonists/cult/cult_structures.dm

```diff
@@ -6,6 +6,10 @@
 	anchored = TRUE
 	light_power = 2
 	debris = list(/obj/item/stack/sheet/runed_metal = 1)
+	/// Length of the cooldown between uses.
+	var/use_cooldown_duration = 5 MINUTES
```

Comment:
you moved `use_cooldown_duration` down a level but not `COOLDOWN_DECLARE(use_cooldown)`

though if you ask me you could totally make the altar an item_dispenser subtype I think it has the ability to support that